internal/wire/ietf_header_test.go

```diff
@@ -133,7 +133,7 @@ var _ = Describe("IETF draft Header", func() {
 
 			It("reads the Key Phase Bit", func() {
 				data := []byte{
-					0x20 ^ 0x1,
+					0x10 ^ 0x40 ^ 0x20,
```

Comment:
It's a bit surprising to see an XOR here. Why not a normal OR?

Reply:
We're using XOR everywhere in these tests, and even in the header writing code. I can change that to OR everywhere if you prefer.

Reply:
It obviously doesn't make a difference, but XOR is unusual for this use case, which is what tripped me up.